### ISSUE TITLE
Midpoint Review - Accessibility Feedback - IVC Forms, 10-10d

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#76492]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Thank you for the thorough accessibility annotations!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** When there's helper text that provides definitions or context for users to understand questions, make sure it's associated with the `input` so that it's announced to screen reader users tabbing through the form. There are a bunch of places where this could be an issue, but I'll highlight Section 2 - Page 9 since it's easy to talk about. The "Enter this information for your sponsor..." paragraph above the inputs is technically accessible, but it's likely to be missed by screen reader users. This deep into a form flow, they're not going to be having their screen reader announce everything on the page, they'll be hitting [tab] to move through each input.
   
   The approaches that seem to work best are:
   - If it's superfluous text and isn't communicating anything necessary for understanding the questions, no action needed. (But also if it's totally superfluous it may not be needed?)
   - If it can be appropriately scoped to one input, include it in the `hint` prop on that input component. The `hint` text is associated with the input with `aria-describedby`.
   - When appropriate to the content, put it in an [additional info component](https://design.va.gov/components/additional-info). The text isn't associated with any of the inputs, but the additional info toggle will be a tab stop so screen reader users should encounter it on their way through the page.
   - If it applies to multiple inputs on the page, do something like:
   ```
   <fieldset>
      <legend>
         <h3>Heading text</h3>
         <p>Some helper text</p>
      </legend>
      [all the inputs]
   </fieldset>
   ```
   The helper text can be styled to look like regular text, but since it's inside the `legend` it will be announced alongside the heading the first time you tab into an input in the fieldset.
   
   Other approaches that we've tried either have screen reader compatibility weirdness or aren't practical with the VA forms system and our web components.
- [ ] **Should:** Allison mentioned alert fatigue on VA.gov, which I'll echo here as a potential cognitive load issue. Getting hit with a lot of things that look visually important can make it harder to parse and identify the things that are actually more pressing. But you all know your product and users better than us, so you can make the judgement on what's critical and what could be downgraded to regular text.
- [ ] **Must (ish):** I have some concerns about some of the info alerts messing with the heading structure on the page, eg. Section 2 - Page 4 and Section 2 - Page 5. Since there's no concept in HTML for "closing" the scope of a heading, a screen reader user navigating the page by heading will find the address or phone number fields under the heading "Resources available" --- which isn't really describing those inputs, it's just supposed to be describing the content of the alert.
   
   I think this would be a WCAG violation if you're playing by the book. Does it actually matter? Most screen reader users are probably just going to be tabbing through and not navigating these pages by heading. But people do unexpected things and they *might*, so it's probably best to try to have a clean heading structure as much as possible. Some possible things to explore:
   - Placement of the alert (after inputs vs before inputs)
   - Adding additional H4 headings to sufficiently delineate the actual form inputs (either visible headings or `sr-only` headings)
   - Downgrading some alerts to regular text, hint text, or additional infos that don't have a heading associated with it. (See alert fatigue above!)

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the Collaboration Cycle initiative epic

**Comment by syd-hoeper** [comment]
Feedback incorporate in this file: https://www.figma.com/file/UmAtr3ULQEInMXfNFwP0g0/10-10d-User-Testing-Prep?type=design&node-id=4%3A64769&mode=design&t=fzOojyFGMdhTn3KU-1
